crates/hyperdrive-math/src/long/targeted.rs

```diff
@@ -215,12 +219,12 @@ impl State {
     /// r'(x) = \frac{-p'(x)}{t \cdot p(x)^2}
     /// ```
     ///
-    /// We return `$-r'(x)$` because negative numbers cannot be represented by FixedPoint.
+    /// We return `$-r'(x)$` because negative numbers cannot be represented by FixedPoint<U256>.
```

Comment:
I did a find and replace for `FixedPoint` to `FixedPoint<U256>` at the start, so there will be some comments like this that aren't true anymore, but I figure those can be addressed in follow-up PRs along with the code the comment refers to.